frontend/src/components/admin-page/edit-user.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from 'react';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { ButtonColor, ButtonStyleType, ButtonType, EditUserPayloadKey, InputColor, InputType, UserSex, UserType } from 'common/enums';
+import styles from './styles.module.scss';
+import { IEditUserPayload, IUser } from 'common/interfaces';
+import { validationEditUser } from 'validation-schemas';
+import { Button, DateInput, Select, TextInput } from 'components/common';
+import { createOptions } from 'helpers';
+
+interface IProps {
+  user: IUser;
+  func: (userData: IEditUserPayload) => void;
```

Comment:
Bad name
You use this type a lot. It would be better to create a own type for it